src/websocket/Connection.js

```diff
@@ -57,6 +59,26 @@ module.exports = class Connection extends EventEmitter {
         return this.streams.map((s) => s.toString())
     }
 
+    addOngoingResend(resend) {
+        this.ongoingResends.add(resend)
+    }
+
+    removeOngoingResend(resend) {
+        this.ongoingResends.delete(resend)
+    }
+
+    getOngoingResends() {
+        return new Set(this.ongoingResends)
+    }
+
+    markAsDead() {
+        this.alive = true
```

Comment:
`this.dead` or `this.alive`? maybe `isAlive` or `isDead`?

Reply:
Reanemd to `this.dead`, I forgot to properly rename when refactoring names. 😬